crates/hotshot/src/traits/networking/web_server_network.rs

```diff
@@ -911,9 +927,11 @@ impl<TYPES: NodeType + 'static, NetworkVersion: StaticVersionType + 'static>
                 }
 
                 // Cancel old, stale tasks
-                task_map
-                    .prune_tasks(view_number, ConsensusIntentEvent::CancelPollForVIDDisperse)
-                    .await;
+                if view_number > 2 {
+                    task_map
+                        .prune_tasks(view_number, ConsensusIntentEvent::CancelPollForVIDDisperse)
+                        .await;
+                }
```

Comment:
Why only for view_number > 2?

(or wait -- is this a web server thing? if so nevermind)

Reply:
Yeah it's some webserver weirdness, we can get by without it probably but for now I'll keep it in